install.sh

```diff
@@ -430,7 +442,7 @@ pushd .
 
   # Build library
   if [[ "${build_relocatable}" == true ]]; then
-    ${cmake_executable} ${cmake_common_options} ${cmake_client_options} \
+    CXX=$compiler ${cmake_executable} ${cmake_common_options} ${cmake_client_options} \
```

Comment:
it doesn't make a difference but would be nice to get the syntax uniform with ${compiler}